certified-connectors/1Me Corporate/Readme.md

```diff
@@ -0,0 +1,37 @@
+# 1Me Corporate
+1Me is the easiest and fastest way to share your contact information.
+With 1Me, you can have an unlimited number of contact cards. You can choose which contact information to include on each card, and then choose who to share each card with.
+This is best for separating your business and personal contact information
+
+
+## Publisher: 1Me
+ 
+
+## Prerequisites
+You will need the following to proceed:
```

Comment:
@MICHAEL-MAURICE is it possible to provide links to the page where users can get the API key?